docs/getting-started-guides/clc.md

```diff
@@ -87,15 +97,13 @@ guide to install the requirements and install the script.
   source credentials.sh
 ```
 
-
-
 ## Cluster Creation
 
 To create a new Kubernetes cluster, simply run the kube-up.sh script. A complete
 list of script options and some examples are listed below.
 
 ```shell
-CLC_CLUSTER_NAME=[name of kubernetes cluster]
+export CLC_CLUSTER_NAME=[name of kubernetes cluster]
 cd ./adm-kubernetes-on-clc
```

Comment:
Previously docs said to go into adm-kubernetes-on-clc/ansible. This should be `cd ../adm-kubernetes-on-clc` or full cloned path